pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -1,9 +1,50 @@
-{ stdenv, fetchgit, fetchFromGitHub, fetchFromGitLab, cmake, pkgconfig, makeWrapper, python27, python37, retroarch
-, alsaLib, fluidsynth, curl, hidapi, libGLU, gettext, glib, gtk2, portaudio, SDL, SDL_net, SDL2, SDL2_image, libGL
-, ffmpeg_3, pcre, libevdev, libpng, libjpeg, libzip, udev, libvorbis, snappy, which, hexdump
-, miniupnpc, sfml, xorg, zlib, nasm, libpcap, boost, icu, openssl
-, buildPackages }:
-
+{ stdenv
```

Comment:
Can you revert this please?

Reply:
Done, but at some point we'll have to reflow it.

Reply:
I agree with you but not probably not one per line. :)